lib/resolveStepDefinition.js

```diff
@@ -118,9 +118,15 @@ function resolveStepArgument(argument, exampleRowData, replaceParameterTags) {
 
     return new DataTable(argumentWithAppliedExampleData);
   }
-  if (argument.type === "DocString") {
+  
+  if (argument.type === "DocString" && exampleRowData) {
+    return replaceParameterTags(exampleRowData, argument.content);
+  }
+
+  if (argument.type === "DocString"&& !exampleRowData) {
```

Comment:
It would be nice to add a code formatter like Prettier in a commit hook :)

Reply:
we do have prettier and a precommit hook that runs it through eslint --fix. :)